packages/components/pie-chip/src/index.ts

```diff
@@ -116,8 +116,8 @@ export class PieChip extends LitElement implements ChipProps {
             'c-chip': true,
             [`c-chip--${variant}`]: true,
             'c-chip--selected': isSelected,
-            'c-chip--dismissible': showCloseButton,
-            'c-chip--disabled': disabled,
+            dismissible: showCloseButton,
```

Comment:
I guess this should also be like this, for consistency? Sorry for being annoying 😄 
```suggestion
            'is-dismissible': showCloseButton,
```